eng/pipelines/runtime-manual.yml

```diff
@@ -499,11 +500,6 @@ jobs:
         testRunNamePrefixSuffix: Mono_$(_BuildConfig)
         scenarios:
         - wasmtestonnodejs
-        condition: >-
-          or(
-          eq(variables['librariesContainsChange'], true),
-          eq(variables['monoContainsChange'], true),
-          eq(variables['isFullMatrix'], true))
```

Comment:
Why don't we want these conditions anymore?

Reply:
In my opinion, there is no other place which would run on this platform every time. Also this is `runtime-manual` pipeline. So it's already run with intention of getting more. I'm prepared to change it if it doesn't make sense.

Reply:
We agreed with Radek to put it back and make it consistent with other ones in the same file.
@steveisok `eq(variables['isManualOrIsNotPR'], true),` is this always true in this file ? Was is the idea behind it ? Or is that just copy&paste like in my case ?